src/ngx_http_lua_socket_udp.c

```diff
@@ -170,7 +170,7 @@ ngx_http_lua_socket_udp_setpeername(lua_State *L)
     ngx_url_t                    url;
     ngx_int_t                    rc;
     ngx_http_lua_loc_conf_t     *llcf;
-    ngx_udp_connection_t        *uc;
+    ngx_resolver_connection_t    *uc;
```

Comment:
Style: the identifier names should be aligned up vertically. I think the `ngx-releng` tool in the nginx-devel-utils repos automatically catches coding style issues like this:

https://github.com/openresty/nginx-devel-utils

Will you please run this tool and fix any coding style issues? Thanks!